src/ObjectUtils/PdbFileLlvm.cpp

```diff
@@ -214,6 +241,117 @@ PdbFileLlvm::PdbFileLlvm(std::filesystem::path file_path, const ObjectFileInfo&
                           llvm::toString(std::move(error)))};
     }
   }
+  return outcome::success();
+}
+
+void LoadDebugSymbolsFromPublicSymbolStream(
+    llvm::pdb::PublicsStream& public_symbol_stream, llvm::pdb::SymbolStream& symbol_stream,
+    llvm::FixedStreamArray<llvm::object::coff_section>& section_headers,
+    ObjectFileInfo object_file_info, std::vector<SymbolInfo>* symbol_infos,
```

Comment:
We know that `ObjectFileInfo` contains a single primitive field now, but still, it should be passed as const-ref.
Same in the other function.